tests/routes/test_document_families.py

```diff
@@ -89,22 +86,23 @@ def test_documents_family_slug_returns_correct_json(
     assert json_response["import_id"] == "CCLW.family.1001.0"
     assert json_response["title"] == "Fam1"
     assert json_response["summary"] == "Summary1"
-    assert json_response["geography"] == "GBR"
+    assert json_response["geography"] == "Other"
     assert json_response["category"] == "Executive"
     assert json_response["status"] == "Published"
     assert json_response["published_date"] == "2019-12-25T00:00:00Z"
     assert json_response["last_updated_date"] == "2019-12-25T00:00:00Z"
 
-    assert len(json_response["metadata"]) == N_METADATA_KEYS
-    assert json_response["metadata"]["keyword"] == ["Energy Supply"]
+    # TODO
```

Comment:
Sort this out

Reply:
What specifically do we need to sort out here?